pages/linux/nsenter.md

```diff
@@ -0,0 +1,17 @@
+# nsenter
+
+> Run a new command in a running process' namespace.
+> Particularly useful for docker images or chroot jails.
+> Homepage: <https://github.com/jpetazzo/nsenter/>.
+
+- Run command in existing processes network namespace:
+
+`nsenter -t {{pid}} -n {{command}} {{command_arguments}}`
+
+- Run a new command in an existing processes ps-table namespace:
+
+`nsenter -t {{pid}} -p {{command}} {{command_arguments}}`
+
+- Run command in existing processes network namespace:
+
+`nsenter -t {{pid}} -n {{command}} {{command_arguments}}`
```

Comment:
This example is the same as the first example 🤔 Should this be removed?

Reply:
Well that's no good! Let's replace it with a different namespace.